index.bs

```diff
@@ -5264,6 +5275,9 @@ given |navigable| and |navigation status|:
 
 1. [=Resume=] with "<code>navigation committed</code>", |navigation id|, and |navigation status|.
 
+1. For each |session| in [=active BiDi sessions=], [=delete collected response bodies=]
```

Comment:
by this point I believe the navigation request that loaded the document has already happened and we want to retain it. If we really want to follow the CDP model we should key the network data by document.

Reply:
Is the response already completed by that time? In any case, adding a reference to the document sounds fine to me I almost wanted to include it in the initial design.

Reply:
I think the headers are read and the body starts being read in parallel. Not having our network hooks in the fetch spec makes it a bit more difficult to cross-check but I think using document's navigation ID would be more resilient (especially if we might be moving the collection to various hooks).

Reply:
I'm having trouble with this part so far.

AFAICT Document's navigation id get set back to null once the load is done (step 9.7 of https://html.spec.whatwg.org/#the-end:concept-document-navigation-id). This means that we won't be able to store the navigation id for all requests.

Storing the document itself is not great either, because the document is normally only created after the response for the document's URL started arriving. 

My current thinking is to store the navigable's ongoing navigation (which is reused as the Document's during-load navigation id), and on navigation committed clear all responses which either:
- don't have any navigation id
- have another navigation id than the navigables ongoing navigation

It might work, but it feels a bit flaky.

Reply:
We are no longer linking navigable/navigations to collected bodies, so closing this thread.